File: co.edu.uniandes.fixml/test/co/edu/uniandes/fixml/src-gen/test5/Java/Instrmt.java
package object;



// Class for tag Instrmt
public class Instrmt {

  String Sym;
  String ID;
  String IDSrc;
  String CFI;
  String MMY;
  String Mat;
  String Strk;
  String StrkCcy;
  String Mult;


  // default constructor
  public Instrmt ( String Sym, String ID, String IDSrc, String CFI, String MMY, String Mat, String Strk, String StrkCcy, String Mult ) {
    this.Sym = Sym;
    this.ID = ID;
    this.IDSrc = IDSrc;
    this.CFI = CFI;
    this.MMY = MMY;
    this.Mat = Mat;
    this.Strk = Strk;
    this.StrkCcy = StrkCcy;
    this.Mult = Mult;
  
  }


  public Instrmt ( ) {   
  
this.Sym = "AOL";
this.ID = "KW";
this.IDSrc = "J";
this.CFI = "OCASPS";
this.MMY = "20031122";
this.Mat = "2003-11-22T00:00:00";
this.Strk = "47.50";
this.StrkCcy = "USD";
this.Mult = "100";
  
  }
      

} 

File: co.edu.uniandes.fixml.toCSharp/test/text2.xml/Instrmt.cs


// Class for tag Instrmt
class Instrmt {

	string _Sym;
	string _ID;
	string _IDSrc;
	string _CFI;
	string _MMY;
	string _Mat;
	string _Strk;
	string _StrkCcy;
	string _Mult;
	
	
	// default constructor
	
	public Instrmt ( ) {   
		this._Sym = "AOL";
		this._ID = "KW";
		this._IDSrc = "3";
		this._CFI = "OCASPS";
		this._MMY = "20031122";
		this._Mat = "2003-11-22T00:00:00";
		this._Strk = "47.50";
		this._StrkCcy = "USD";
		this._Mult = "100";
		
		
	}
	
	public Instrmt ( string Sym, string ID, string IDSrc, string CFI, string MMY, string Mat, string Strk, string StrkCcy, string Mult ) {
		this._Sym = Sym;
		this._ID = ID;
		this._IDSrc = IDSrc;
		this._CFI = CFI;
		this._MMY = MMY;
		this._Mat = Mat;
		this._Strk = Strk;
		this._StrkCcy = StrkCcy;
		this._Mult = Mult;
		
	}
	
	
} 